src/Agent.Worker/TestResults/NunitResultReader.cs

```diff
@@ -8,46 +8,22 @@
 
 namespace Microsoft.VisualStudio.Services.Agent.Worker.TestResults
 {
-    public class NUnitResultReader : AgentService, IResultReader
+    internal class NUnitResultsXmlReader
```

Comment:
How will you get the extension dlls to the agent? There is no way to manage such extensions and thats why we have coded this in. 
WF